core/metrics.go

```diff
@@ -0,0 +1,318 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package core
+
+import (
+	"time"
+
+	source_api "k8s.io/heapster/sources/api"
+)
+
+// Stub out for testing
+var timeSince = time.Since
+
+var SupportedMetrics = []SupportedMetric{
```

Comment:
We will support all metrics, not only these mentioned below. Name it as StandardMetrics or so.